_pages/showcase.md

```diff
@@ -0,0 +1,56 @@
+
+We are accepting 10 minute talks from students from year 7 to 12. 
+
+## Important Dates
+- Call for Proposals opens: Tuesday, 1st May 2019
+- Proposal submission deadline: early June 2019
+- Notification of whether proposals have been accepted, not accepted, or waitlisted: mid June, 2019
+- Students whose applications are accepted are invited to attend the entire conference from Friday - Sunday
+
+## Speaker Benefits
+Accepted students will have the opportunity to present their work in front of interested and supportive industry professionals and teachers from other schools. They will be provided with free registration for the specialist events day and both days of the conference main track. Should a speaker also wish to attend the sprints and workshops these will attract a small charge. Attendance at any paid tutorials will also attract a charge at the standard rates.
+
+If you are able to pay for a full registration, then please consider doing so. The funds from registrations contribute not only to conference costs, but also the financial aid program. Every extra dollar makes it easier for someone else to attend.
+
+## Financial Assistance
+PyCon Australia offers a generous financial assistance program, so that some attendees and speakers have some (or in rare cases, all) of their expenses, such as accommodation or flights, reimbursed to them from the conference budget. PyCon Australia strongly encourages people to apply for financial assistance. We don't want to lose a good talk due to financial need.
+
+The full details of the financial aid program from last year are available here. As with previous years, speakers will be given priority for financial aid requests within reasonable limits. Financial assistance requests are evaluated separately and cannot be considered guaranteed even if your talk is accepted.
+
+Some speakers may have attended other conferences where full costs have been met, including travel and accommodation. As a not-for-profit, community-run conference, this is generally not the case at PyCon AU. 
+
+## Code of Conduct
+PyCon Australia 2019 has a Code of Conduct [https://2019.pycon-au.org/conduct/](https://2019.pycon-au.org/conduct/) which outlines expectations of behaviour and appropriate presentation content. The goal of the code of conduct is to facilitate a safe, welcoming and inclusive environment. Presenters are required to be familiar with and abide by the code of conduct. If you have any questions about the suitability of any aspect of your proposed talk or require any additional information, please email us at [program@pycon-au.org](mailto:program@pycon-au.org).
```

Comment:
The program email this year is  speakers@2019.pycon-au.org

Reply:
done